Frontend/unhoused-outreach-ui/src/App.css

```diff
@@ -58,4 +58,14 @@ table tr:last-child td:last-child {
 
 h3 {
   text-align: center;
-}
\ No newline at end of file
+}
+
+.accordion {
```

Comment:
do we want to have individual style sheets for different components? Or just keep adding to App.css?

Reply:
moved to separate style sheets